runtime/common/BaseRemoteRESTQPU.h

```diff
@@ -457,6 +470,17 @@ class BaseRemoteRESTQPU : public cudaq::QPU {
         throw std::runtime_error("Could not successfully apply quake-synth.");
     }
 
+    // Delay combining measurements for backends that cannot handle
+    // subveqs and multiple measurements until we created the emulation code.
+    auto combineMeasurements =
+        passPipelineConfig.find("combine-measurements") != std::string::npos;
+    if (emulate && combineMeasurements) {
+      std::regex combine("(.*),([ ]*)combine-measurements(.*)");
```

Comment:
This is complicated and subtle enough that I think it merits a `cudaq::info` message when this is happening (even as part of this PR). Do we also plan to circle back to this? Ideally the emulation code path and true REST server API would not deviate behaviors in this way (at least not for the long term).

Reply:
done, thanks!

Reply:
nit: this will also not work if we ever have a future pass named `combine-measurements-with-a-random-suffix` or `this-is-a-random-prefix-to-combine-measurements`. If this is only temporary code, then it probably doesn't matter, but that's just more motivation to circle back to this at some point soon.